src/input.js

```diff
@@ -84,10 +84,8 @@ export class Input {
     // mouse events.
     if (this.hasTouch_) {
       this.hasMouse_ = !this.hasTouch_;
-      if (!this.boundOnMouseMove_) {
-        this.boundOnMouseMove_ =
-          /** @private {!Function} */ (this.onMouseMove_.bind(this));
-      }
+      this.boundOnMouseMove_ =
+        /** @private {!Function} */ (this.onMouseMove_.bind(this));
```

Comment:
check-types fails if I don't have it:
found   : function (Event): (Promise|undefined)
required: (function ((Event|null)): ?|null)